lib/completions-jsx.coffee

```diff
@@ -77,6 +77,18 @@ globalAttributes: [
 	{ name: 'data-'}
 	{ name: 'aria-'}
 	{ name: 'style'}
+        { name: 'accesskey'}
+        { name: 'contenteditable'}
+        { name: 'contextmenu'}
+        { name: 'dir'}
+        { name: 'draggable'}
+        { name: 'dropzone'}
+        { name: 'hidden'}
+        { name: 'lang'}
+        { name: 'spellcheck'}
+        { name: 'tabindex'}
```

Comment:
Some of these need to be camelCased, like `tabindex` ➡️  `tabIndex`. See https://facebook.github.io/react/docs/dom-elements.html for a full description.

Reply:
In fact, it looks like there's an exhaustive list including the casing here: https://facebook.github.io/react/docs/dom-elements.html#all-supported-html-attributes